priv/repo/migrations/20211103061515_add_expired_matches_table.exs

```diff
@@ -0,0 +1,17 @@
+defmodule T.Repo.Migrations.AddExpiredMatchesTable do
+  use Ecto.Migration
+
+  @opts [on_delete: :delete_all, type: :uuid]
+
+  def change do
+    create table(:expired_matches, primary_key: false) do
+      add :match_id, :uuid, null: false
+      add :user_id, references(:users, @opts), null: false
+      add :with_user_id, references(:users, @opts), null: false
+
+      timestamps(updated_at: false)
+    end
+
+    create unique_index(:expired_matches, [:match_id, :user_id])
```

Comment:
Why not `[user_id, match_id]`? I think the table is queried by user_id to get matches, not the other way around.